test/e2e/http/server.test.ts

```diff
@@ -23,6 +23,7 @@ describe("http/server", () => {
   let server: DaprServer;
   const mockBindingReceive = jest.fn(async (_data: object) => console.log("mockBindingReceive"));
   const mockPubSub = jest.fn(async (_data: object) => null);
+  const mockPubSubWithHeaders = jest.fn(async (_data: object, _headers: object) => null);
```

Comment:
Can you please add this under `beforeEach` as `mockPubSubWithHeaders.mockClear();` (both http and grpc)?